src/otx/core/model/entity/base.py

```diff
@@ -40,8 +40,10 @@ def __init__(self, num_classes: int) -> None:
         super().__init__()
 
         self._label_info = LabelInfo.from_num_classes(num_classes)
+        self.num_classes = num_classes
```

Comment:
Can we get `num_classes` from `self.label_info`?
```suggestion
        @property
        def num_classes(self) -> int:
             return len(self.label_info)
```

Reply:
Are we sure that there is always gonna be `self.label_info`?
Updated, take a look.

Reply:
> Are we sure that there is always gonna be `self.label_info`? 
Yes